### ISSUE TITLE
"flutter doctor" is absolutely fine but "flutter run" doesnot work and produces following error

### ISSUE THREAD

**Issue by engr-zeeshan** [flutter/flutter#26862]
```
Running preremove grass.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log 
The system cannot find the path specified.
The syntax of the command is incorrect.
Running preremove opencl.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log 
The system cannot find the path specified.
The syntax of the command is incorrect.
Running preremove pyqt5.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log 
The system cannot find the path specified.
The syntax of the command is incorrect.
Running preremove python3-certifi.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log 
The system cannot find the path specified.
The syntax of the command is incorrect.
Running preremove python3-chardet.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log 
The system cannot find the path specified.
The syntax of the command is incorrect.
Running preremove python3-core.bat... Zeeshan\AppData\Local\Temp\QGIS-OSGeo4W-3.4.3-1-preremove.log
.......
.......
.......
.......
```

![screenshot_1](https://user-images.githubusercontent.com/6541601/51492113-75e83280-1dd2-11e9-9387-09ba08a7604d.png)

**Comment by zoechi** [comment]
That doesn't even look like is is related to Flutter.
Perhaps your Python install is broken and Flutter run a Python command.

**Comment by engr-zeeshan** [comment]
I also confused what was going on. I tried again by re-installing python but the error remained.

1- Prior to this error, I was facing another error related to "adb" path and I found that issue was because of "space" in path. The problem was solved by changing path of android sdk.

- After changing path (that doesn't contain "space") of android sdk , problem 1 was resolved but then I faced the problem mention in this issue above. This was also due to "space" within the path of flutter project that i was trying to "run". By changing path of project, everything is working fine now.

Regards.
